### DISCUSSION TITLE
NZB stopped working (the way it usually does)...server-mode

### DISCUSSION THREAD

**Discussion by maussimo010** [nzbgetcom/nzbget#306] [Q&A]
I appreciate any help in advance!
I've been running NZB for years via Docker on a Synology NAS with no issues at all. A day or two ago, int he messages section, it says "server-mode". When I drag an NZB file into it, nothing happens. No speed limit, time remaining, nothing. I've also tried updating using Portainer, and it says the image can't be pulled, and I think thats because I just found out that it's no longer being supported, so there is no longer a current version on the repository I was using, which was linuxserver.io. Since NZBget worked so well, I rarely updated it because it "just worked". Now since I have found out that v24 is out n0w, and I am still on v21, I'd like to update my version and see if it fixes my issue with server mode. Before I figure out how to do that, does anyone have a fix for the server mode issue (I don't even know if that IS my issue; it's just that it's not working properly now and I don't ever remember seeing server mode there before). I GREATLY appreciate any help or advice anyone has to offer! Thank you!

https://imgur.com/a/oLvFBP8

**Comment by dnzbk** [comment]
The problem is that you are trying to add an NZB file to the Messages tab, which is not designed for that. Go to the Downloads tab (the first one with the downward pointing arrow) and try adding the NZB file again. Also at the Downloads tab, there is an Add (with “+” icon) button that can be used to add NZB files directly.

**Reply by maussimo010** [reply]
I must not have explained myself clearly. I am definitely not dropping .nzb files into the messages tab, as I had said in the first line that I have been using NZBget for at least 5-7 years now. What I was stating about the messages tab is that it says server mode in it, which it has never said before, and that .nzb files were not downloading, that I could visibly see anyway. I know precisely where and how to add .nzb files though, and I am with you on stating that is not onto the messages window. Thank you for reading though.

**Reply by dnzbk** [reply]
Actually, the Messages tab has been saying “server-mode” since at least NZBGet v16.0 which was released around 9 years ago, so that’s likely not the case.
Could you share the nzbget.log file, located in your download directory? This would help me to troubleshoot the problem.

Please consider using the newer version of NZBGet available [here](https://nzbget.com/download/) which has many improvements and bug fixes.
These [instructions](https://github.com/nzbgetcom/nzbget/blob/develop/docs/SYNOLOGY.md#synocommunity-package) for Synology users can be helpful for you.

**Reply by maussimo010** [reply]
I would love nothing but more to update to the latest version, but I’m not 100% confident in my ability to do so without messing things up and then having no working version at all.

**Reply by maussimo010** [reply]
It seems that I have accidentally updated to v24.1 and aside from a few setup issues, it seems to be working well. Download speeds are not the greatest by any means, and I dont know if there was a change in coding anywhere that would improve this. On my old version v21, I would usually get a burst of around 50-60MB/s for a minute or so, then it would slow down to around 20-30ish most of the time. I've tried varying the amount of connections, but I haven't gotten anything higher than 10-12MB/s. I'm saying for a 1000 down, and here is what I just got from a speedtest running from my laptop, so not hardwired:  DOWNLOAD Mbps
234.18  UPLOAD Mbps 40.19. My host is newshosting and I have been using them for years, and I am open to suggestions if anyone tells me these guys suck. I'm also reading about cipher, and that is wayyy over my head, and I have a sneaking suspicion that may be the issue. All I know is that my NZBget, Radarr, and Sonarr are running via Docker on my Synology, and not on the Synology directly. I tried the RC4-MD5 cipher and was given this in return: Could not select cipher for TLS: error:0A0000B9:SSL routines::no cipher match so I removed it completely.

**Reply by dnzbk** [reply]
That's a really annoying situation, sorry for that. The download speed issues are my top priority. 
I would greatly appreciate your assistance in investigating this problem.

1) newshosting recommends using 30 or 100 connections, depending on your plan. Could you try these specific values to see if they improve the download speed?

2) I’ve added some improvements in this direction only in NZBGet v24.2 (currently in testing) that address speed spikes. Before my changes I could observe a bug where download speeds would occasionally drop significantly, sometimes even to zero, for brief periods - could you try installing the testing version to see if it resolves the issue?

Thanks in advance.

**Reply by maussimo010** [reply]
I don't mind trying, but I need to become familiar with what im doing, and back up my settings to make sure i dont lose anything since I finally got v24 working. Let me have the weekend and I can probably do it Saturday morning or maybe Friday night. 

As for testing the 30-100, my results are at 30 connections is about 15-20MB/s down, mostly around 18-20MB/s on average...at 100 connections was between 25-32MB/s, hovering around 28-30MB/s during download of a 5gb file.

**Reply by maussimo010** [reply]
I did not change my paths, but I’m guessing the paths that were there when I got v24 up and running are not MY paths, as I now have downloaded files I cannot find that are in the downloads/completed folder…where is the completed folder located when it’s left at its default setting?

**Reply by dnzbk** [reply]
Do I understand correctly that the speed at 100 connections is about the same as the v21 version?

Paths can be found and changed in `SETTINGS`->`PATHS`.
You can also backup your settings in `SETTINGS`->`SYSTEM`.

**Reply by maussimo010** [reply]
I know where the settings are, but doesn’t tell me where on my system my files are that I’ve downloaded. Obvi it says downloads/completed, but where is that? I don’t see it on my NAS.

**Reply by luckedea** [reply]
@maussimo010 Can you check Container Manager -> Container -> nzbget -> General and show what you have under Volume.

**Reply by maussimo010** [reply]
I don't think I have a container manager?  I've looked. When I open docker on my Synology, the only options I have on the left sidebar are:

<img width="248" alt="Screenshot 2024-07-12 at 11 45 03 AM" src="https://github.com/user-attachments/assets/a76a0bab-58cb-45b1-bb1c-b6bb16d9743c">

I've never used Docker Compose either, because that's now how I learned to install Docker or how to install/update docker apps on my Synology. I started using Portainer about a year ago to update the apps I currently have installed in Docker.

Just realized I have never updated docker, and I am in the process of updating it now to v24. 

Just found out my DSM for the NAS needs to be 7.2, current is 7.1, and the NAS is showing that there is no update available, so doing a manual DSM update, THEN after that's installed I can install the latest version of Container Manager, which requires DSM 7.2

Legs are tired from jumping through all these hoops.

Ok new container manager installed.

on my OLD version of nzbget, v21, this is what I had for Volume:

<img width="836" alt="Screenshot 2024-07-12 at 12 40 27 PM" src="https://github.com/user-attachments/assets/a862c3f8-73f4-445b-b09d-a49e648972ae">


on the NEW version of nzbget, v24, this is what it shows under volume:

<img width="569" alt="Screenshot 2024-07-12 at 12 38 08 PM" src="https://github.com/user-attachments/assets/693ea361-9da6-42b8-9f40-187cfbf84e67">

As you can see, I have NO volume with v24, and it does not point to the media folders I need it to on my NAS.

**Reply by maussimo010** [reply]
So I have read that with the new container manager update (which is now up and running for me), that you can't change the original image, and you must duplicate it to make changes to it. I can see where to add paths to my files I want to make available to nzbget, but I am a bit over my head here cause I havant learned this software yet, and I reallllly don't want to take a chance of messing something up. I'd appreciate some help on getting my paths squared away, and to know if these changes affect sonarr and radarr as well, and if so how to make the necessary changes to them as well.

**Reply by maussimo010** [reply]
Guess who's back...back again.

Ok, so with the new container manager installed, I finally realize what the project tab is for, and alas, what docker compose is!

I copypasta'ed from the documentation page, created the necessary folders, and the build went fine. Now my issue is that after I have plugged my host providers server info in, for the two that I use, I am getting an ":Could not resolve hostname news.newshosting.com: Error -3 - Try again" message, so I still can't test to see if my missing downloaded files issue is resolved. I get this same error on both of the servers I test. Also, if I try to check for an update, I get "Check failed. Please try again later." So, I think something is wrong with my network settings inside NZBget so that it can't phone out, and I am guessing this is related to docker/container manager thing to where it says somewhere "use same host as", but I haven't found that part yet in project/compile area to be able to change it of even examine it.

log from container manager:


2024/07/13 01:47:03 | stdout | [INFO] nzbget 24.1 server-mode
2024/07/13 01:30:48 | stdout | [ERROR] Could not resolve hostname nzbget.com: Error -3 - Try again
2024/07/13 01:30:23 | stdout | [ERROR] Could not resolve hostname nzbget.com: Error -3 - Try again
2024/07/13 01:29:28 | stdout | [ERROR] Could not resolve hostname nzbget.com: Error -3 - Try again
2024/07/13 01:28:32 | stdout | [WARNING] Request received on port 6789 from 172.18.0.1, but username () or password invalid
2024/07/13 01:28:28 | stdout | [ERROR] Could not resolve hostname nzbget.com: Error -3 - Try again
2024/07/13 01:28:24 | stdout | [WARNING] Request received on port 6789 from 172.18.0.1, but username () or password invalid
2024/07/13 01:27:46 | stdout | [WARNING] Request received on port 6789 from 172.18.0.1, but username () or password invalid
2024/07/13 01:27:41 | stdout | [WARNING] Request received on port 6789 from 172.18.0.1, but username () or password invalid
2024/07/13 01:23:31 | stdout | [INFO] nzbget 24.1 server-mode
2024/07/13 01:23:31 | stdout | [INFO] Reloading...
2024/07/13 01:21:38 | stdout | [INFO] nzbget 24.1 server-mode
2024/07/13 01:17:54 | stdout | [INFO] nzbget 24.1 server-mode

This is what my network section has, and for driver, it says "bridge"

<img width="477" alt="Screenshot 2024-07-13 at 1 54 00 AM" src="https://github.com/user-attachments/assets/61d9afaa-6a32-44b8-9fac-8c769a373822">


Help! 

I am stoked that I have come this far and gotten to update all these things that I didn't really realize needed updating!

UPDATE: Duplicated the container that I built in project manager. Found the network settings. Changed them to same as host. Tested both servers and they were successful. Downloaded a file. unpacked and ended up in the complete folder as it should. 

GREAT SUCCESS!

so if this is working now, do I need to change anything with Sonarr/Radarr?
